src/en/suryascans/src/eu/kanade/tachiyomi/extension/en/suryascans/GenzToons.kt

```diff
@@ -42,6 +44,16 @@ class GenzToons :
         }
     }
 
+    override fun pageListParse(document: Document): List<Page> {
+        val script = document.select("#pages > script").joinToString("\n") { it.data() }
+        val realCdnUrl = CDN_URL_REGEX.find(script)?.groupValues?.get(1)?.takeIf { it.startsWith("http") }
+            ?: "$baseUrl/uploads/"
+        return document.select("#pages > img")
+            .mapIndexed { index, img ->
+                Page(index, document.location(), realCdnUrl + img.attr("uid"))
+            }
+    }
+
     override fun setupPreferenceScreen(screen: PreferenceScreen) {
```

Comment:
What do you think about moving these settings(_**Display paid chapters**_) to the theme. So some issues related to the paid chapter can be closed.

Like this one: #4999

Reply:
yeah but should be another PR imo